chemprop/uncertainty/calibrator.py

```diff
@@ -68,13 +70,59 @@ def apply_calibration(self, preds, uncs) -> Tensor:
 
 @UncertaintyCalibratorRegistry.register("platt")
 class PlattCalibrator(UncertaintyCalibrator):
-    def calibrate(self, preds, uncs, targets, mask) -> Tensor:
-        ...
-        return
-
-    def apply_calibration(self, preds, uncs) -> Tensor:
-        ...
-        return
+    """
+    A calibration method for classification datasets based on the Platt scaling algorithm.
+    As discussed in https://arxiv.org/abs/1706.04599 and Platt, J. (2000). Probabilistic outputs for
+    support vector machines and comparison to regularized likelihood methods. In A. Smola, P. 
+    Bartlett, B. Schölkopf, & D. Schuurmans (Eds.), Advances in large margin classifiers. Cambridge:
+    MIT Press.
+    In Platt's paper, he suggests using the number of positive and negative examples in the dataset
+    used to train the model to adjust the value of target probabilities used to fit the parameters. 
+    """
+
+    def calibrate(self, preds, targets, mask, training_targets: None | Tensor = None) -> Tensor:
```

Comment:
this return type annotation is incorrect.

Reply:
That is my bad. I will fix it in the [v2/uncertainty](https://github.com/chemprop/chemprop/tree/v2/uncertainty) branch.

Reply:
I just added a commit that fixes it for all them before I saw your comment here.

Reply:
fixed!